src/search.js

```diff
@@ -1,111 +0,0 @@
-const _ = require('underscore');
-
-const config = require('./config');
-const bsonUrlEncoding = require('./utils/bsonUrlEncoding');
-
-/**
- * Performs a search query on a Mongo collection and pages the results. This is different from
- * find() in that the results are ordered by their relevancy, and as such, it does not take
- * a paginatedField parameter. Note that this is less performant than find() because it must
- * perform the full search on each call to this function.
- *
- * @param {MongoCollection} collection A collection object returned from the MongoDB library's
- *    or the mongoist package's `db.collection(<collectionName>)` method. This MUST have a Mongo
- *    $text index on it.
- *    See https://docs.mongodb.com/manual/core/index-text/.
- * @param {String} searchString String to search on.
- * @param {Object} params
- *    -query {Object} The find query.
- *    -limit {Number} The page size. Must be between 1 and `config.MAX_LIMIT`.
- *    -fields {Object} Fields to query in the Mongo object format, e.g. {title :1}.
- *      The default is to query ONLY _id (note this is a difference from `find()`).
- *    -next {String} The value to start querying the page. Defaults to start at the beginning of
- *      the results.
- */
-module.exports = async function(collection, searchString, params) {
-  if (_.isString(params.limit)) params.limit = parseInt(params.limit, 10);
```

Comment:
This line is no longer present. Should we keep it to make we don't break anything with JS code?